release-notes/opensearch-release-notes-2.16.0.md

```diff
@@ -0,0 +1,874 @@
+
+
+### Opensearch k-NN
+
+
+* Bump faiss commit to 33c0ba5 [#1796](https://github.com/opensearch-project/k-NN/pull/1796)
+
+
+## REFACTORING
+
+
+### Opensearch ML Common
+
+
+* Change multimodal connector name to bedrock multimodal connector (#2672)[https://github.com/opensearch-project/ml-commons/pull/2672]
+
+
+### Opensearch Reporting
+
+
+* Register system index descriptors through SystemIndexPlugin.getSystemIndexDescriptors ([#1009](https://github.com/opensearch-project/reporting/pull/1009))
+
+
+### SQL
+
+
+* Change DataSourceType from enum to class ([#2746](https://github.com/opensearch-project/sql/pull/2746))
+* Fix code style issue ([#2745](https://github.com/opensearch-project/sql/pull/2745))
+* Scaffold async-query-core and async-query module ([#2751](https://github.com/opensearch-project/sql/pull/2751))
+* Move classes from spark to async-query-core and async-query ([#2750](https://github.com/opensearch-project/sql/pull/2750))
+* Exclude integ-test, doctest and download task when built offline ([#2763](https://github.com/opensearch-project/sql/pull/2763))
+* Abstract metrics to reduce dependency to legacy ([#2768](https://github.com/opensearch-project/sql/pull/2768))
+* Remove AsyncQueryId ([#2769](https://github.com/opensearch-project/sql/pull/2769))
+* Add README to async-query-core ([#2770](https://github.com/opensearch-project/sql/pull/2770))
+* Separate build and validateAndBuild method in DataSourceMetadata ([#2752](https://github.com/opensearch-project/sql/pull/2752))
+* Abstract FlintIndex client ([#2771](https://github.com/opensearch-project/sql/pull/2771))
+* Fix statement to store requested langType ([#2779](https://github.com/opensearch-project/sql/pull/2779))
+* Push down OpenSearch specific exception handling ([#2782](https://github.com/opensearch-project/sql/pull/2782))
+* Implement integration test for async-query-core ([#2785](https://github.com/opensearch-project/sql/pull/2785))
+* Fix SQLQueryUtils to extract multiple tables ([#2791](https://github.com/opensearch-project/sql/pull/2791))
+* Eliminate dependency from async-query-core to legacy ([#2792](https://github.com/opensearch-project/sql/pull/2792))
+* Pass accountId to EMRServerlessClientFactory.getClient ([#2822](https://github.com/opensearch-project/sql/pull/2822))
+* Register system index descriptors through SystemIndexPlugin.getSystemIndexDescriptors ([#2817](https://github.com/opensearch-project/sql/pull/2817))
+* Introduce SparkParameterComposerCollection ([#2824](https://github.com/opensearch-project/sql/pull/2824))
+
+
+## NON-COMPLIANT
+
+
+## FEATURES/ENHANCEMENTS
+
+
+### Opensearch Dashboards Notifications
+
+
+* side navigation changes for notifications ([#222](https://github.com/opensearch-project/dashboards-notifications/pull/222))([#225](https://github.com/opensearch-project/dashboards-notifications/pull/225))
+* Backport/backporting to 2.x ([#223](https://github.com/opensearch-project/dashboards-notifications/pull/223))
+
+
```

Comment:
1. Need to talk to the OSD Notification team on their formatting being wrong, should separate feature and enhancement to two different categories.
2. Confirm with them then put the section to the corresponding category.

Thanks.

Reply:
Sure @peterzhuamazon.